changelog/db.changelog-1.66.xml

```diff
@@ -0,0 +1,117 @@
+    </insert>
+    <insert tableName="autoHoldCategories">
+      <column name="autoHoldCategoryId" value="31" />
+      <column name="schemeId" value="3" />
+      <column name="name" value="Awaiting debt enrichment" />
+    </insert>
+    <insert tableName="autoHoldCategories">
+      <column name="autoHoldCategoryId" value="42" />
+      <column name="schemeId" value="3" />
+      <column name="name" value="Manual ledger hold" />
+    </insert>
+    <insert tableName="autoHoldCategories">
+      <column name="autoHoldCategoryId" value="44" />
+      <column name="schemeId" value="4" />
+      <column name="name" value="Awaiting debt enrichment" />
+    </insert>
+    <insert tableName="autoHoldCategories">
+      <column name="autoHoldCategoryId" value="55" />
+      <column name="schemeId" value="4" />
+      <column name="name" value="Manual ledger hold" />
+    </insert>
+    <insert tableName="autoHoldCategories">
+      <column name="autoHoldCategoryId" value="57" />
+      <column name="schemeId" value="5" />
+      <column name="name" value="Awaiting debt enrichment" />
+    </insert>
+    <insert tableName="autoHoldCategories">
+      <column name="autoHoldCategoryId" value="68" />
+      <column name="schemeId" value="5" />
+      <column name="name" value="Manual ledger hold" />
+    </insert>
+    <insert tableName="autoHoldCategories">
+      <column name="autoHoldCategoryId" value="85" />
+      <column name="schemeId" value="6" />
+      <column name="name" value="Awaiting debt enrichment" />
+    </insert>
+    <insert tableName="autoHoldCategories">
+      <column name="autoHoldCategoryId" value="112" />
+      <column name="schemeId" value="7" />
+      <column name="name" value="Awaiting debt enrichment" />
+    </insert>
+    <insert tableName="autoHoldCategories">
+      <column name="autoHoldCategoryId" value="116" />
+      <column name="schemeId" value="6" />
+      <column name="name" value="Manual ledger hold" />
+    </insert>
+    <insert tableName="autoHoldCategories">
+      <column name="autoHoldCategoryId" value="117" />
+      <column name="schemeId" value="7" />
+      <column name="name" value="Manual ledger hold" />
+    </insert>
+    <insert tableName="autoHoldCategories">
+      <column name="autoHoldCategoryId" value="122" />
+      <column name="schemeId" value="12" />
+      <column name="name" value="Manual ledger hold" />
+    </insert>
+    <insert tableName="autoHoldCategories">
+      <column name="autoHoldCategoryId" value="124" />
+      <column name="schemeId" value="12" />
+      <column name="name" value="Awaiting debt enrichment" />
+    </insert>
+  </changeSet>
+  <changeSet author="Sam Plackett" id="4">
+    <addForeignKeyConstraint baseColumnNames="autoHoldCategoryId" baseTableName="autoHolds" constraintName="fk_autoHolds_autoHoldCategoryId" referencedColumnNames="autoHoldCategoryId" referencedTableName="autoHoldCategories" />
+  </changeSet>
```

Comment:
Does this need a FK reference to schemes?

Reply:
autoHolds doesn't but autoHoldCategories certainly does.